packages/salesforcedx-vscode-core/src/commands/forceDataSoqlQuery.ts

```diff
@@ -63,7 +63,7 @@ export class GetQueryAndApiInputs
         query = document.getText(editor.selection);
       }
     }
-    query = query!.replace('[', '').replace(']', '');
+    query = query!.replace('[', '').replace(']', '').replace('\n', ' ');
```

Comment:
What happens on Windows that use \r\n as the line break?

Reply:
Good question.  I'm developing on windows and it seems to work, but may be because I have my git and editorconfig set to enforce `\n` even on windows.  Easy enough to fix.

```suggestion
    query = query!.replace('[', '').replace(']', '').replace(/(\r\n|\n)/g, " ");
```

Reply:
@vazexqi you can see the original submission here had single quotes... not sure why I changed that when I switched to regex!